metrics.js

```diff
@@ -0,0 +1,28 @@
+const client = require('prom-client');
+const Registry = client.Registry;
+const register = new Registry();
+
+// Create a new metric counter
+const counter = new client.Counter({
+	  name: 'metric_name',
+        help: 'metric_help'
+});
+require('prom-client').collectDefaultMetrics({ register });
+
+// New recordcount 
+function recordcount() {
```

Comment:
I'd say you can remove this counter and just have the default metrics for now.

Reply:
@david-martin happy to remove the counter it was just a test. As the other npm module above and will test it on a cluster on see. Happy to go with the alternative after the test.